### ISSUE TITLE
Accessibility

### ISSUE THREAD

**Issue by unclegcb** [thinkshout/bene#221]
## Overview
Goal:
Make all Bene sites WCAG 2.0 compliant

Some work has been done in this area already. Next steps:

- Scan live bene sites, and a default bene site, for accessibility issues

- Create task list to resolve those issues that are core to Bene (Any project-specific issues should be added to those client site accessibility tasks.)

- Create an accessibility contact form & add default page to the build, link in footer, etc.

**Comment by unclegcb** [comment]
@komejo where are we with merging the accessibility work done already on your open source day? Is it merged? Do we need to worry about translating it to the new version of the base theme?

**Comment by komejo** [comment]
Sorry for letting this languish - my fixes were merged in on Jun 13.

**Comment by unclegcb** [comment]
awesome, sounds like it should all be integrated with the new work by default then.

That means we can work on this ticket actively to reach the desired level of compliance.
